src/Network/Send/kRO/RagexeRE_2020_03_04a.pm

```diff
@@ -0,0 +1,123 @@
+#########################################################################
+#  OpenKore - Packet sending
+#  This module contains functions for sending packets to the server.
+#
+#  This software is open source, licensed under the GNU General Public
+#  License, version 2.
+#  Basically, this means that you're allowed to modify and distribute
+#  this software. However, if you distribute modified versions, you MUST
+#  also distribute the source code.
+#  See http://www.gnu.org/licenses/gpl.html for the full license.
+########################################################################
+#bysctnightcore
+package Network::Send::kRO::RagexeRE_2020_03_04a;
+
+use strict;
+use base qw(Network::Send::kRO::Ragexe_2018_11_14c);
+use Globals; 
+use Data::Dumper;
+use Log qw(error debug message);
+use I18N qw(stringToBytes);
+
+sub new {
+	my ($class) = @_;
+	my $self = $class->SUPER::new(@_);
+	
+	my %packets = (
+		'0ACF' => ['master_login', 'a4 Z25 a32 a5', [qw(game_code username password flag)]],
```

Comment:
its `password_rijndael`

Reply:
rathena dont check it.

no need password_rijndael

Reply:
we can send the same package as the client

Reply:
fixed : 8dae9df